meteor/client/ui/Settings/helpers/show-style-variant-configuration-verifier.ts

```diff
@@ -0,0 +1,78 @@
+import { ShowStyleVariant } from '../../../../lib/collections/ShowStyleVariants'
+import { ShowStyleBase } from '../../../../lib/collections/ShowStyleBases'
+import {
+	ConfigItemValue,
+	ConfigManifestEntry,
+	ConfigManifestEntrySelectFromColumn,
+	ConfigManifestEntryType,
+} from '@sofie-automation/blueprints-integration'
+
+export class ShowStyleVariantConfigurationVerifier {
+	public isBlueprintConfigurationSelectedFromBaseInvalidForAllVariants(
+		showStyleVariants: ShowStyleVariant[],
+		showStyleBase: ShowStyleBase,
+		blueprintConfigManifest: ConfigManifestEntry[]
+	): boolean {
+		return showStyleVariants.some((variant) =>
+			this.isBlueprintConfigurationSelectedFromBaseInvalid(variant, showStyleBase, blueprintConfigManifest)
+		)
+	}
+
+	public isBlueprintConfigurationSelectedFromBaseInvalid(
+		showStyleVariant: ShowStyleVariant,
+		showStyleBase: ShowStyleBase,
+		blueprintConfigManifest: ConfigManifestEntry[]
+	): boolean {
+		const configuredVariantEntries: [string, any][] = Object.entries(showStyleVariant.blueprintConfig)
+		return configuredVariantEntries.some(([configurationName, configuredValue]) =>
+			this.hasConfigurationSelectedInvalidValueFromShowStyleBase(
+				configurationName,
+				configuredValue,
+				showStyleBase,
+				blueprintConfigManifest
+			)
+		)
+	}
+
+	private hasConfigurationSelectedInvalidValueFromShowStyleBase(
```

Comment:
Is `Selected` important in the name? Is `Configuration` not given from the context? Could `hasInvalidConfigurationValueFromShowStyleBase` or `hasInvalidValueFromShowStyleBase` suffice?

Reply:
Yes and no.
It was suppposed to be an indication of what specify configuration we were testing, but I don't think I achieved that very well.
I have updated it to one of your suggestions